src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -371,9 +371,15 @@ configurationRegistry.registerConfiguration({
 			'default': 9
 		},
 		'explorer.autoReveal': {
-			'type': 'boolean',
-			'description': nls.localize('autoReveal', "Controls whether the explorer should automatically reveal and select files when opening them."),
-			'default': true
+			'type': ['boolean', 'string'],
+			'enum': [true, false, 'highlightNoScroll'],
+			'default': 'autoReveal.on',
+			'enumDescriptions': [
```

Comment:
I think this enum descriptions and the default are wrong. Default should be true.

Reply:
Nice catch, I missed that line. Committed again.